cypress/integration/anton.sample/todoSample.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Managing of todo list
+  As a user
+  I want to use todo list functionality
+  So I can plan my activities
+
+  Background:
+    When I add items via localStorage
+    When I open todo list page
+    Then I see that header is shown
+    Then I see that "item1" item is presented
+    Then I see that footer is shown
+
+  Scenario: Add new item into the list
+    When I type "Test1" into input field and press Enter
+    Then I see that "Test1" item is presented
+    Then I see that number of left items is "4"
```

Comment:
Better to avoid hard-coding and magic numbers when this is possible. You can save number of items added in localStorage and compare new value. Step could look like: `Number of active items should increase by 1`